.env

```
@@ -27,3 +27,5 @@ REACT_ENABLED_APPS=""
 
 REACT_INJECTED_HEAD=""
 REACT_INJECTED_HTML=""
+
+REACT_APP_STORE_INDEX_URL=https://raw.githubusercontent.com/ohcnetwork/care_apps_registry/refs/heads/main/index.json
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)